### DISCUSSION TITLE
What library is used to generate the `nuxt dev` network QR Code?

### DISCUSSION THREAD

**Discussion by quentint** [nuxt/nuxt#23239] [Questions] [answered]
Hey everyone, the title says it all 😉 

I'm looking for the library/module that is used to render the QR Code visible when running the `nuxt dev` command in the CLI. I've searched the code and the release notes but didn't find anything (I'm afraid it lives in another repo). Can anyone help?

Thanks!

**Comment by leopiccionia** [comment, marked as answer]
It's [uqr](https://github.com/unjs/uqr). If I remark correctly, it's called by [listhen](https://github.com/unjs/listhen) (inside [nitro](https://github.com/unjs/nitro)).

**Reply by quentint** [reply]
That's a perfect answer, thanks!
